### ISSUE TITLE
measure accuracy and confusion matrix

### ISSUE THREAD

**Issue by abhi-84** [intel/scikit-learn-intelex#214]
Hi
In daal4py, can we calculate accuracy score of model along with confusion matrix, precision, recall, F1 score etc ? How? Any example?

**Comment by mHash1m** [comment]
I'd also like to know how to calculate accuracy using daal4py

**Comment by PivovarA** [comment]
Hi @abhi-84 . 
As far as I know, this problem can be solved using **sklearn confusion_matrix**, which takes *y_true* and *y_prediction* as parameters.
Or the question is how to do that by using daal4py itself?

**Comment by abhi-84** [comment]
Yes.Need to know how to do it using daal4py itself?

**Comment by PivovarA** [comment]
For now, we recommend users to use sklearn for measurement accuracy
